controllers/__tests__/User.test.js

```diff
@@ -171,15 +171,34 @@ describe("User queries", () => {
   });
 
   describe("Change password", () => {
-    test("should return rowCount == 1 if change successfully", async () => {
+    test("should return rowCount == 1 if change successfully with old password", async () => {
```

Comment:
Can we add a test against the new password to make sure new password has been set?